src/ORM/DataObject.php

```diff
@@ -2146,6 +2147,14 @@ public function getFrontEndFields($params = null)
         return $untabbedFields;
     }
 
+    /**
+     * {@inheritdoc}
+     */
```

Comment:
AFAIK this would be the same result as not having a doc block at all, so may as well omit it

Reply:
Good point, will remove 😄